.studio/chef-server-collection

```diff
@@ -70,6 +71,19 @@ ssl_verify_mode        :verify_none
 client_key             '${chef_server_test_admin_key_path_path}'
 validation_client_name 'test-validator'
 validation_key         '${chef_server_test_validator_path}'
+EOH
+
+  cat << EOH > ${chef_server_policyfile_rb_path}
+log_location           STDOUT
+chef_server_url        'https://${chef_server_hostname}/organizations/${chef_server_test_org_name}'
+node_name              '${chef_server_test_node_name}'
+ssl_verify_mode        :verify_none
+client_key             '${chef_server_test_admin_key_path_path}'
+validation_client_name 'test-validator'
+validation_key         '${chef_server_test_validator_path}'
+policy_group           'test_policy_group'
+policy_name            'examplecb'
+use_policyfile         true
```

Comment:
Adding a new client.rb file to add the policyfile information.

Reply:
How did this change the behavior? `policy_group` should come from the CLI argument to `chef push` and `policy_name` should be in the `Policyfile.rb` file. IIRC `use_policyfile` should only change the behavior of the `chef generate` command, but you can force it with `-P` (and policyfile is the default now).

Reply:
My guess was that this change added the policyfile to the local run of chef-client on line 178. The rest of the references to this file could be replace by the client.rb file above. 
`CHEF_LICENSE=accept-no-persist chef-client -c ${chef_server_policyfile_rb_path}`

Reply:
Ah, I see it now. We generally suggest to users to set these in the first boot json so I forgot they are referenced in the policyfile-or-not logic in Chef Client. For the use case here it's fine to use the config file though. 👍